docs/en/latest/router-radixtree.md

```diff
@@ -289,6 +291,56 @@ $ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f
 }'
 ```
 
+We can verify GraphQL matches in the following two ways:
+
+1. GraphQL query strings
+
+    ```shell
+    $ curl -H 'content-type: application/graphql' -X POST http://127.0.0.1:9080/graphql -d '
+    query getRepo {
+        owner {
+            name
+        }
+        repo {
+            created
+        }
+    }'
+    ```
+
+2. JSON format
+
+    ```shell
+    $ curl -H 'content-type: application/json' -X POST http://127.0.0.1:9080/graphql --data '{"query": "query getRepo { owner {name } repo {created}}"}'
+    ```
+
+Next, try a GET request.
+Let's first configure a route like this:
+
+```shell
+$ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -i -d '
+{
+    "methods": ["GET"],
+    "uri": "/_graphql",
+    "vars": [
+        ["graphql_operation", "==", "query"],
+        ["graphql_name", "==", "getRepo"],
+        ["graphql_root_fields", "has", "owner"]
+    ],
+    "upstream": {
+        "type": "roundrobin",
+        "nodes": {
+            "39.97.63.215:4000": 1
+        }
+    }
+}'
+```
```

Comment:
Why directly reuse the routing configuration of POST? You can add GET to the `methods` of your route configuration to support both GET and POST requests.

Reply:
It can be done as you say, let me try it.

Reply:
Finished it! 
Thanks for a review @shuaijinchao